mlflow/utils/autologging/__init__.py

```diff
@@ -1023,219 +1048,207 @@ def safe_patch_function(*args, **kwargs):
         while exceptions thrown from other parts of `patch_function` are caught and logged as
         warnings.
         """
-        if _is_testing():
-            preexisting_run_for_testing = mlflow.active_run()
-
-        original = gorilla.get_original_attribute(destination, function_name)
-
-        # If the autologging integration associated with this patch is disabled,
-        # call the original function and return
-        if autologging_is_disabled(autologging_integration):
-            return original(*args, **kwargs)
-
-        # Whether or not to exclude auto-autologged content from content explicitly logged via
-        # `mlflow.start_run()`
-        exclusive = get_autologging_config(autologging_integration, "exclusive", False)
```

Comment:
This logic and the logic below it was right-intended and wrapped in the `set_mlflow_events_and_warnings_behavior_globally` / `set_non_mlflow_warnings_behavior_for_current_thread` context managers. The logic itself remains unchanged (with the exception of 3 lines here: https://github.com/mlflow/mlflow/pull/4173/files#r600105840)